### ISSUE TITLE
Enable users to subscribe to important Kyma notifications

### ISSUE THREAD

**Issue by varbanv** [kyma-project/kyma#18708]
**Description**

Provide a mechanism for end users to subscribe for important notifications related to their clusters. 
It should be a list of emails that the SRE team can fetch and integrate with the notification system.

**Reasons**

Users should be informed about important events related to their clusters. That includes upgrades and disruptions, but also changes in underlying services and infrastructure. 

**Attachments**
- [Ensure that 24x7 contact exist for each Kyma runtime](https://github.tools.sap/kyma/backlog/issues/5067)

**Links**
- https://github.wdf.sap.corp/pages/sl-hybrid/ans/notifications-scenario/onboarding-and-integration/onboard-to-cockpit-scenario/#scenario-overview

**Progress**:
- we need to organise a meeting with SRE to discuss the topic and when we can have it

**Comment by zhoujing2022** [comment]
- Can we align with CF to design a general solution for a unified runtime?

- We can maybe add an additional step in the BTP cockpit during creation of new cluster:
  - Include a bullet point to highlight this: https://help.sap.com/docs/btp/sap-business-technology-platform/kyma-s-modular-approach?version=Cloud#release-notes or maybe something about the SPC and Cloud available subscriptions. (@ebensom what do you think?)
  - Add a required field to collect the administrator’s email address.
  - Store these email addresses in the SPC instead of Kyma, if it's possible.

**Comment by zhoujing2022** [comment]
- As discussed, we still need to clarify the PED due to the involvement of personal data processing. I will check with another team in FP to see if they know how to handle this.
- @ebensom will create a PoC to explore how the automation can be set up with SPC.
- According to @PK85, the change in KEB is minimal—only one parameter will to be added.
- This issue can only be addressed after the above points have been clarified. The work will likely begin in Q3 or Q4 of 2025.

**Comment by zhoujing2022** [comment]
@szwedm 
In my opinion, we already have existing communication channels for release notes and availabilities. Therefore, this notification doesn’t need to duplicate those messages. We should keep these notifications to a minimum and use them only for important alerts, such as:

- Upcoming breaking changes
- High-risk periods
- Major maintenance windows

**Comment by szwedm** [comment]
@zhoujing2022 I agree, use cases for notifications in the platform I've shown were merely examples.

**Comment by PK85** [comment]
Unassignign myself, no priority for now
